carbon-design-system/carbon-for-ibm-dotcom
# fix(link-with-icon): adjust left icon placement style & clean up

### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7074

### Description

- This PR fixes the issue where the `left` icon placement is not aligning to the left-hand side.

### Changelog

**Changed**

- Adjust styling in the shared `_link-with-icon.scss` style and clean up duplicate styling.

### Testing Instruction

- Visit the "Link with Icon" component
- Adjust the "Icon Position (icon-placement)" knob to `left`
- Ensure that the placement of the icon is in alignment with the design specs.
